Course_Part_04/src/Assignment19.java

```diff
@@ -10,6 +10,7 @@ public static void main(String[] args) {
 
     private static double multiplyDouble(double... numbers) {
```

Comment:
de variabele `sum1` kan gewoon `sum` heten of mooier misschien nog `result` omdat het niet een optelling representeert. Over de scope van een variabele gaan we het nog hebben, maar de `sum` variabele in de main heeft niets te maken met de variabele in de methode, vandaar kunnen ze beide dezelfde naam hebben.